test/simulator/moves/electricterrain.js

```diff
@@ -70,6 +70,7 @@ describe('Electric Terrain', function () {
 		battle = BattleEngine.Battle.construct();
 		battle.join('p1', 'Guest 1', 1, [{species: "Jolteon", ability: 'voltabsorb', moves: ['electricterrain', 'rest']}]);
 		battle.join('p2', 'Guest 2', 1, [{species: "Pidgeot", ability: 'keeneye', moves: ['doubleedge', 'rest']}]);
+		battle.seed = [0, 0, 0, 0];
```

Comment:
It's not even crit Double Edge. It's near-maxdamage crit Double Edge, with like ~35% of OHKO if it crits.